modules/ROOT/pages/to-describe-an-asset.adoc

```diff
@@ -12,90 +12,90 @@ Each minor version of an asset has its own tags, categories, custom fields, and
 
 In Exchange, open the asset list and select the asset. Exchange shows the asset portal.
 
-== Edit Asset Portal Name
+== Edit Asset Portal Information
 
-. Select the asset portal name or the pen icon after the name.
-. Enter the new name.
-. (Optional) To cancel the changes, select the cancel icon.
-. To save the new name, press the enter or return key on your keyboard, or select the check icon.
+You can change the asset icon, name, description, contact name, and contact email. 
 
-NOTE: Before the August 2019 release, the versions of an asset could have different names. After the August 2019 release, a new name applies to all versions of the asset.
+. Click the edit pen icon in the asset portal.
+. To change the icon, click the edit pen icon, select an image file, then click *Open*.
+. To change the asset portal name, enter the new asset portal name.
+. To change the description, enter a new asset description in the *Description* field.
+. To change the contact name or email, enter a new contact name and email in the corresponding field.
+. To cancel the changes, click the *X* icon.
+. To save the changes, click the *check* icon.
 
+NOTE: Before the August 2019 release, the versions of an asset could have different names or icons. After the August 2019 release, a updated asset information applies to all versions of the asset.
 
-== Edit Asset Icon
+== Add and Remove Categories
 
-. Select the pen icon beside the asset icon.
-. Select an image.
-. (Optional) To remove an asset icon, select the cancel icon beside the asset icon.
+When searching for assets, categories can be used to filter the search results. 
 
-NOTE: Before the August 2019 release, the versions of an asset could have different icons. After the August 2019 release, a new asset icon applies to all versions of the asset.
+. In the asset details pane, click *Add categories*. 
```

Comment:
The "Add categories" is only shown when no categories are present. If a category was already applied it would read Categories (1) and the process to add a category would differ slightly. I'm not sure if we want to specify that as well.